lib/pages/settings_pages/settings_page.dart

```diff
@@ -134,17 +125,17 @@ class _SettingsPageState extends State<SettingsPage> {
             SizedBox(
               height: 10,
             ),
-            SettingsMenuWidget(
-                hashtag == null
-                    ? MyLocalizations.of(context, 'enable_auto_hashtag')
-                    : MyLocalizations.of(context, 'disable_auto_hashtag'), () {
-              _manageHashtag();
-            }),
-            SizedBox(
-              height: 35,
-            ),
-            SizedBox(
-              height: 60,
+            Container(
+              padding: EdgeInsets.only(bottom: 12.0, top: 12.0),
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(5),
+                color: Colors.white,
+                border: Border.all(color: Colors.black.withOpacity(0.1))
+              ),
+              child: ListTile(
+                title: Style.body('Auto-tag reports'),
```

Comment:
localise: Use string_id: `auto_tagging_settings_title`

Reply:
I've added the 2 localise string_id